### PR TITLE
script for rolling update milvus's componet which installed by helm

### PR DESCRIPTION
**Pull request by locustbaby**
1. only support simple upgrade now,will upgrade the component in a safe order
2. won't keep old version information and don't support rollback now
3. the topology is hard-coded now
4. will patch instance's configmap and turn on standby mode
5. will patch instance's update strategy to rollingUpdate mode

Signed-off-by: Sheldon <chuanfeng.liu@zilliz.com>

### CODE REVIEW

File: deployments/upgrade/rollingUpdate.sh [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+# entrypoint
+while getopts "n:i:t:w:o:" OPT; do
+    case $OPT in
+    n) NAMESPACE=$OPTARG ;;
+    i) INSTANCE=$OPTARG ;;
+    t) TARGET_VERSION=$OPTARG ;;
+    w) IMAGE_TAG=$OPTARG ;;
+    o) OPERATION=$OPTARG ;;
+    *) echo "Unkonwen parameters" ;;
```
Comment (zhuwenxing) [inline deployments/upgrade/rollingUpdate.sh:172]: Just find a typo, Unkonwen -->Unknown

Comment (locustbaby) [reply deployments/upgrade/rollingUpdate.sh:172]: thanks a lot, fixed

File: deployments/upgrade/rollingUpdate.sh [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+function Usage() {
+    echo "
+
+    [Warning]: Based on kubectl
+    -n [namespace]                        default: [default]
+                                          The namespace that Milvus is installed in.
+    -i [instance name]                    [required] milvus_instance: The name of milvus instance.
+    -t [target_version]                   target_version: The milvus target version.
+    -w [image_tag]                        image_tag: The new milvus image tag.
+    -o [operation]                        operation: The operation: migrate/rollback.
```
Comment (zhuwenxing) [inline deployments/upgrade/rollingUpdate.sh:154]: In the current implementation， the operation only supports `update`.

This usage info display is not consistent with README.
image

**Review by LoveEachDay — APPROVED** [review]
/lgtm

**Pull request merged**